andela/ah-commando-frontend
# #167190400 comment functionality on articles

#### What does this PR do?
Implements comment functionality on articles

#### Description of Task to be completed?
- Users can view all comments an article has
- Users can add a comment to the list of existing comments

#### How should this be manually tested?
- View a single article
- Click on the comment icon and ensure you are authenticated
- When redirected to the comment page you will view all comments and can add a new comment
#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
#167190400
